lib/vanagon/project.rb

```diff
@@ -289,10 +289,14 @@ def has_conflicts?
     # will return nil
     #
     # @param [string] name of service to grab
-    # @return [@component.service obj] specific service
+    # @return [@component.service obj] specific service, or array of services
+    #         if there's more than one
     def get_service(name)
```

Comment:
I might be missing something, but I don't see any changes to how we're using `get_service` in this PR, but I would expect there to be some adjusting based on whether there's more than one or not.

Reply:
get_service isn't used in vanagon anywhere. It has minimal use in projects (for example, in agent it's only used in windows packaging), and will only need change if you're using it and setting multiple services.

Reply:
ooh okay